src/components/LoginDialog.jsx

```diff
@@ -38,16 +43,62 @@ export default function LoginDialog() {
       open={showLoginDialog}
       onClose={closeDialog}
       aria-labelledby="responsive-dialog-title">
-      <DialogTitle id="responsive-dialog-title">
-        Login using your account or email.
-      </DialogTitle>
-      <DialogContent>
-        <DialogContentText>
-          You can login using your existing account to authenticate and log in
-          to our web-app. You can create an account if you do not want to use
-          your existing account to interact with our app.
-        </DialogContentText>
-        <StyleFirebaseAuth uiConfig={uiConfig} firebaseAuth={firebase.auth()} />
+      <DialogContent sx={{ p: 0 }}>
+        <Grid container spacing={2}>
+          <Grid
+            item
+            xs={12}
+            sm={6}
+            sx={{ color: 'background.paper', bgcolor: 'primary.main' }}>
+            <DialogTitle id="responsive-dialog-brand">
+              <Typography sx={{ color: 'background.paper' }}>
+                DreamScholars
+              </Typography>
+            </DialogTitle>
+
+            <DialogTitle id="responsive-dialog-welcome">
+              <Typography variant="h4" sx={{ color: 'background.paper' }}>
+                Welcome.
+              </Typography>
+            </DialogTitle>
+
+            <DialogContentText sx={{ color: 'background.paper', p: 3 }}>
+              <Typography paragraph sx={{ color: 'background.paper' }}>
+                DreamScholars provides scholarships for all students regardless
+                of status.
+              </Typography>
+              <Typography paragraph sx={{ color: 'background.paper' }}>
+                Join our community &amp;
+              </Typography>
+              <Typography paragraph sx={{ color: 'background.paper' }}>
+                Get access to scholarship submissions and other cool features
+                soon.
+              </Typography>
+            </DialogContentText>
+          </Grid>
+
+          <Grid item xs={12} sm={6}>
+            <Box sx={{ textAlign: 'right' }}>
```

Comment:
I suggest moving this up one parent so that the X appears on the blue box on mobile.

You might need to change its position so that it doesn't end up pushing content down.